addons/survey/static/src/scss/survey_form.scss

```diff
@@ -0,0 +1,251 @@
+.o_survey_header {
+    height: 50px;
+}
+.o_survey_main {
+    min-height: calc(100% - 105px);
+    padding-top: 16px;
+}
+
+.o_survey_question_text_box,
+.o_survey_question_date,
+.o_survey_question_datetime,
+.o_survey_question_numerical_box {
+    border: 0px;
+    border-radius: 0px;
+    border-bottom: 2px solid $o-brand-primary;
+    background-color: transparent !important;
+    font-weight: bold;
+    color: $o-brand-primary !important;
+    &:disabled {
+        border-color: grey;
+        color: black !important;
+        border-bottom: 2px solid grey;
+    }
+    &:focus {
+        border-color: $o-brand-primary;
+    }
+
+    &::-webkit-scrollbar {
+      width: 10px;
+    }
+    &::-webkit-scrollbar-thumb {
+      background: grey;
+      border-radius: 10px;
+    }
+    &::-webkit-scrollbar-thumb:hover {
+      background: white;
+    }
```

Comment:
I avoid prefixed classes like this.
You're aware this will not work on firefox right?

Reply:
What is the purpose ? Isn't it any generic solution instead ? Maybe ask to some smart css people as it seems a bit tricky.

Reply:
well the issue is only in print mode where the scroll bar is super ugly.
There is a scroll bar in print mode as the widget is not initialized on this print template.
and the auto resize is not done, consequently.
Either I remove this crappy css and too bad for the ugly scroll bar in print mode, either I have to make a tiny widget for survey_print to only autoresize all textarea.